__tests__/e2e/helpers/verifySubmission.ts

```diff
@@ -91,33 +161,33 @@ const TABLE_HANDLER = {
 /**
  * Gets the title of a field as it is displayed in a response.
  * @param {E2eFieldMetadata} field field used to create and fill form
- * @param {boolean} isInJson Whether the title is within the JSON data for email submissions
  * @param {FormResponseMode} formMode form response mode
  * @returns {string} the field title displayed in the response.
  */
 const getResponseTitle = (
   field: E2eFieldMetadata,
-  isInJson: boolean,
   formMode: FormResponseMode,
 ): string => {
-  if (field.fieldType === 'table') {
+  // MyInfo fields
+  if (isMyInfoableFieldType(field) && field.myInfo) {
+    if (field.myInfo.verified) return `[MyInfo] ${field.title}`
+    return field.title
+  }
+
+  // Basic fields
+  if (field.fieldType === BasicField.Table) {
     return TABLE_HANDLER.getName(field, formMode)
-  } else if (field.fieldType === 'attachment') {
-    let title = field.title
-    if (formMode === 'email') {
-      title = `[attachment] ${title}`
-    }
-    return title
-  } else {
-    if (isInJson) {
-      if (field.title.startsWith('[verified] ')) {
-        return field.title.substring(11)
-      } else if (field.title.startsWith('[MyInfo] ')) {
-        return field.title.substring(9)
-      }
+  } else if (field.fieldType === BasicField.Attachment) {
```

Comment:
We could change this `else if` clause into `if` since the previous `if` returns.